starkcore/utils/request.py

```diff
@@ -65,18 +64,19 @@ def fetch(host, sdk_version, user, method, path, payload=None, query=None,
         )
     except Exception as exception:
         error = "{}: {}".format(exception.__class__.__name__, str(exception.__context__))
-        if prefix == "Joker":
-            return Response(status=500, content=error, headers=request.headers if "headers" in request else "")
-        raise UnknownError(error)
-
-    response = Response(status=request.status_code, content=request.content, headers=request.headers)
-    if prefix != "Joker":
-        if response.status == 500:
-            raise InternalServerError()
-        if response.status == 400:
-            raise InputErrors(response.json()["errors"])
-        if response.status != 200:
-            raise UnknownError(response.content)
+        response = Response(status=0, content=error, headers={})
+    else:
+        response = Response(status=request.status_code, content=request.content, headers=request.headers)
+
+    if not raiseException:
```

Comment:
faz sentido a gente ter um parametro só pra isso ?? sendo que se for passado algo no prefix significa que o cara ta usando joker ??